config/certmanager/certmanager_role_binding.yaml

```diff
@@ -1,25 +1,39 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: controller-manager-certmanager
+  labels:
+    app.kubernetes.io/created-by: argo-cd
+    app.kubernetes.io/instance: kcp-lifecycle-manager
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: lifecycle-manager
+    app.kubernetes.io/part-of: kcp
+  name: klm-controller-manager-certmanager
   namespace: istio-system
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
-  name: controller-manager-certmanager
+  name: klm-controller-manager-certmanager
```

Comment:
Should this prefix be offered by `namePrefix: klm-` in the kustomization? We shouldn't not add them manually.